highfidelity/hifi
# case 21032: Life cycle pause

https://highfidelity.manuscript.com/f/cases/21032/Quest-fix-issues-with-lifecycle-of-the-app

A hack that prevents the app from releasing JavaHooks during on pause and pushes the release into the OnDestroy step. 


Test:

1. Start app
2. Go to any domain
3. hit the oculus button on your right hand controller
4. In the Quit/resume window, select Resume
The app should resume without any hitches, possibly some slowness for a second while the tasks are caught up.
5. Test tablet, movement, audio , mic
6. Press oculus button again
7. Select quit
8. Open app again.

The app should resume normally.
